<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MLSDialog</class>
 <widget class="QWidget" name="MLSDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MLS smoothing and reconstruction</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 51, 54);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>24</number>
      </property>
      <property name="horizontalSpacing">
       <number>39</number>
      </property>
      <property name="verticalSpacing">
       <number>16</number>
      </property>
      <item row="0" column="0" colspan="4">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="search_radius_label">
          <property name="text">
           <string>Search Radius</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="search_radius">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>2.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.005000000000000</double>
          </property>
          <property name="value">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLabel" name="upsampling_method_label">
              <property name="text">
               <string>Upsampling Method</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>1</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>1</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #444444;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QRadioButton" name="radioButtonnone">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>None</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QRadioButton" name="radioButtonSample">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Sample Local Plane</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="upsampling_radius_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">padding-left:20px;</string>
        </property>
        <property name="text">
         <string>Upsampling Radius</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="upsampling_radius">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>152</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>152</width>
          <height>34</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QDoubleSpinBox
{
	color: #ffffff;
	background-color: #343336;
	border:1px solid #989898;
	padding-left:9px;
	border-radius: 2px;
	min-height:32px;
	max-height:32px;
}
QDoubleSpinBox:disabled
{
	color: #999999;
	background-color: #3F3F3F;
}</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>2.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.005000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="upsampling_step_size_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Upsampling Step</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QDoubleSpinBox" name="upsampling_step_size">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>152</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>152</width>
          <height>34</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QDoubleSpinBox
{
	color: #ffffff;
	background-color: #343336;
	border:1px solid #989898;
	padding-left:9px;
	border-radius: 2px;
	min-height:32px;
	max-height:32px;
}
QDoubleSpinBox:disabled
{
	color: #999999;
	background-color: #3F3F3F;
}</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>2.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="4">
       <widget class="QRadioButton" name="radioButtonRandom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Uniform density random sampling</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="step_point_density_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">padding-left:20px;</string>
        </property>
        <property name="text">
         <string>Step Point Density</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="step_point_density">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>152</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>152</width>
          <height>34</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QSpinBox
{
	color: #ffffff;
	background-color: #343336;
	border:1px solid #989898;
	padding-left:9px;
	border-radius: 2px;
	min-height:32px;
	max-height:32px;
}
QSpinBox:disabled
{
	color: #999999;
	background-color: #3F3F3F;
}</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QRadioButton" name="radioButtonVoxel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Voxel Grid Dilation</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="dilation_voxel_grid_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">padding-left:20px;</string>
        </property>
        <property name="text">
         <string>Dilation Voxel Size</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="dilation_voxel_size">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>152</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>152</width>
          <height>34</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QDoubleSpinBox
{
	color: #ffffff;
	background-color: #343336;
	border:1px solid #989898;
	padding-left:9px;
	border-radius: 2px;
	min-height:32px;
	max-height:32px;
}
QDoubleSpinBox:disabled
{
	color: #999999;
	background-color: #3F3F3F;
}</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>0.500000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="dilation_iterations_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Dilation Iterations</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QSpinBox" name="dilation_iterations">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>152</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>152</width>
          <height>34</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QSpinBox
{
	color: #ffffff;
	background-color: #343336;
	border:1px solid #989898;
	padding-left:9px;
	border-radius: 2px;
	min-height:32px;
	max-height:32px;
}
QSpinBox:disabled
{
	color: #999999;
	background-color: #3F3F3F;
}</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="4">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Storage Path</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>8</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>32</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{padding-left:12px;border-radius: 2px;border: 1px solid #989898;color:#F2F2F2;}
QLineEdit:disabled{color:#838383;border: 1px solid #666666;background-color: #3F3F3F}
</string>
            </property>
            <property name="placeholderText">
             <string>Select the storage path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>32</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{border: 1px solid #555555;
			border-radius: 2px; 
			background: #555555; 
			color: #F2F2F2; }
QPushButton:hover { background-color: #2B2B2B; }
QPushButton:disabled { color: #838383;background-color:#555555;}</string>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
